src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -5,7 +5,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>true</IsPackable>
     <NuspecFile>Microsoft.Health.Fhir.Liquid.Converter.nuspec</NuspecFile>
-    <OrasWinUrl>https://github.com/deislabs/oras/releases/download/v0.8.1/oras_0.8.1_windows_amd64.tar.gz</OrasWinUrl>
+    <OrasWinUrl>https://github.com/deislabs/oras/releases/download/v0.9.0/oras_0.9.0_windows_amd64.tar.gz</OrasWinUrl>
```

Comment:
Do we need to update the oras version in build pipeline?

Reply:
Yes, we need.